src/quo2/screens/list_items/dropdown.cljs

```diff
@@ -0,0 +1,71 @@
+(ns quo2.screens.list-items.dropdown
+  (:require [quo.components.text :as text]
+            [quo.previews.preview :as preview]
+            [quo.react-native :as rn]
+            [quo2.components.list-items.dropdown :as quo2]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]))
+
+(def descriptor [{:label   "Icon"
+                  :key     :icon
+                  :type    :select
+                  :options [{:key   :main-icons/placeholder
```

Comment:
can you add an option of "none" so it's possible to verify the designs here.

Reply:
I think clicking clear on the icon selector removes it